### DISCUSSION TITLE
Limit the number of votes per option/timeslot

### DISCUSSION THREAD

**Discussion by bvqbao** [lukevella/rallly#1388] [Ideas]
Hi,

There was a request for this in 2018: https://github.com/lukevella/rallly/issues/59.

Could this feature (the ability to limit the votes per option) be reconsidered now as Rally is in good shape?
This is a 'missing' feature I found in comparison to Doodle.

This could help schedule meetings between teachers and students for example.

Thanks,

**Comment by Cicatr1x** [comment]
I would highly appreciate this feature as well. For me it would make it possible to schedule half yearly meetings with parents possible via Rallly. Until now i have to use a different system.

**Comment by gitwittidbit** [comment]
+1 from me. I appreciate that the idea for Rallly was different but if Rallly is being promoted as a Doodle alternative, this feature is definitely missing. Just set Rallly up on my own server only to find out that the one feature I need is not implemented :(

Don't get me wrong. Rallly is cool and certainly has its use cases but I automatically assumed that this function would be supported. Didn't even think to double check before installing.
